integration/spring-pulsar/src/main/java/com/example/pulsar/SpringPulsarApplication.java

```diff
@@ -0,0 +1,42 @@
+package com.example.pulsar;
+
+import org.apache.pulsar.client.api.Schema;
+import org.apache.pulsar.common.schema.SchemaType;
+
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.pulsar.annotation.PulsarListener;
+import org.springframework.pulsar.core.PulsarTemplate;
+
+@SpringBootApplication
+public class SpringPulsarApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(SpringPulsarApplication.class, args);
+    }
+
+    @Bean
+    ApplicationRunner runner1(PulsarTemplate<Greeting> pulsarTemplate) {
+        String topic = "graal-topic";
```

Comment:
**[TinyNit]** I think it helps w/ consistency if we make the topic and subscription name prefixes match such as `graalvm-demo-topic`.